src/app/shared/team-radio-switch/team-radio-switch.component.html

```diff
@@ -7,8 +7,11 @@
       [(ngModel)]="valueId"
       #refInput1
     />
-    <img [src]="team1.iconUrl" [alt]="team1.name" />
-    <div class="input-name">{{ team1.name }}</div>
+    <app-team-logo
+      [name]="team1.name"
+      [iconUrl]="team1.iconUrl"
+      [isUnchecked]="refInput2.checked"
```

Comment:
There is semantic inconsistency in a bound property and value you assign to it

Reply:
I don't know if I understood the comment correctly, but I changed `[isUnchecked]="refInput2.checked"` that I think should correct the semantic inconsistency